client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -222,6 +228,65 @@ const CommunitySettings = () => {
 						setDescription(evt.target.value.substring(0, 280).replace(/\n/g, ' '));
 					}}
 				/>
+				<InputField
+					label={
+						<span>
+							Journal Citation
+							<Tooltip
+								content={
+									<span>
+										When filled out, this field will be used as the journal
+										title for
+										<br />
+										issue collections in citations, PDF exports, and Crossref
+										deposits.
+										<br />
+										By default, the Community Title is used.
+									</span>
+								}
+								// @ts-expect-error ts-migrate(2322) FIXME: Type '{ children: Element; content: Element; toolt... Remove this comment to see the full error message
+								tooltipClassName="bp3-dark"
+							>
+								<Icon icon="info-sign" />
+							</Tooltip>
+						</span>
+					}
+					type="text"
+					placeholder="This will show up in citations and BibTex as your journal name"
+					value={citeAs}
+					onChange={(evt) => {
+						setCiteAs(evt.target.value);
+					}}
```

Comment:
The `<InputField>` component currently doesn't account for non-string labels. It renders our label w/ tooltip fine (especially with the CSS changes we made), but the `for` and `id` attributes render like `id="input-[object Object]"` because JS automatically casts the JSX we pass to the `label` prop to the string "[object Object]" when  concatenated with the `"label-"` prefix [here](https://github.com/pubpub/pubpub/blob/master/client/components/InputField/InputField.tsx#L56) and [here](https://github.com/pubpub/pubpub/blob/master/client/components/InputField/InputField.tsx#L83). This introduces an accessibility problem where clicking the label of the "Publisher" input actually focuses the "Journal Citation" input, since the `for` attribute links the label to an `<input>` element.

We'll need to modify the `<InputField>` component to fix this. Taking a look at the `<ImageUpload>` component, it looks like an `htmlFor` attribute was exposed which is used in both the final `for` and `id` props passed to the underlying HTML `<input>` element.

Sorry, I know this is an information-dense comment. Happy to hop on a quick call to discuss if I did a bad job with my explanation.